examples/oneview_storage_volume_attachment_facts.yml

```diff
@@ -78,6 +95,7 @@
         options:
           - paths
       delegate_to: localhost
+      when: currentVersion == '200' and currentVersion == '300'
```

Comment:
i think it should be 'or' instead of 'and'

Reply:
done